### PR TITLE
Comments front end

### PR DESCRIPTION
**Pull request by tammytamayao**
- CRUD front-end comments feature of the app
- connected the front-end to the back-end rails API

### CODE REVIEW

File: src/comments/EditComment.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    
+                <button type="submit">Update Comment</button> <br/>
+                <Link to="/posts">Back to Main</Link>
+    
+              </form>
+        </div>
+    )
+}
+
+export default EditComment;
```
Comment (rafael-parayno) [inline src/comments/EditComment.tsx:87]: add new line

File: src/comments/NewComment.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                    </div>
+        
+                    <button type="submit">Add Comment</button> <br/>
+        
+                  </form>
+            </div>
+    )
+}
+
+export default NewComment;
```
Comment (rafael-parayno) [inline src/comments/NewComment.tsx:70]: add newline

File: src/config/AxiosConfig.tsx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+const baseURL = process.env.REACT_APP_API_ACTIVE+`api/v1`;
+const client = axios.create({
+    baseURL: baseURL
+});
+
+export { baseURL, client }
```
Comment (rafael-parayno) [inline src/config/AxiosConfig.tsx:8]: add newline

**Review by rafael-parayno — CHANGES_REQUESTED** [review]

File: src/config/AxiosConfig.tsx (hunk 3 above)
Comment (tammytamayao) [reply src/config/AxiosConfig.tsx]: done

File: src/comments/NewComment.tsx (hunk 2 above)
Comment (tammytamayao) [reply src/comments/NewComment.tsx]: done

File: src/comments/EditComment.tsx (hunk 1 above)
Comment (tammytamayao) [reply src/comments/EditComment.tsx]: done

**Review by rafael-parayno — APPROVED** [review]

**Pull request merged**